benchmark/reformat_input_gi.py

```diff
@@ -0,0 +1,143 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""
+Reformat input genome files to formats required by given
+GI tool
+=========================================================
+"""
+
+import click
+from os import remove
+from os.path import join
+
+from skbio import Sequence, DNA
+
+
+def reformat_egid(genbank_fp,
```

Comment:
I tried to use skbio to write a GenBank format file. However, EGID had problem parsing this file. I thought part of the reason is that skbio's GenBank file has different leading space pattern comparing to native NCBI GenBank format. For example: `     gene            ` is the native pattern, but skbio does not precisely replicate that (I cannot recall how it looks like).